runtime/src/globals/fetch/response/mod.rs

```diff
@@ -176,5 +176,18 @@ pub mod class {
 			let bytes = self.read_to_bytes().await?;
 			String::from_utf8(bytes).map_err(|e| Error::new(&format!("Invalid UTF-8 sequence: {}", e), None))
 		}
+
+		pub async fn json(&mut self, cx: &Context<'_>) -> Result<*mut JSObject> {
+			let text = self.text().await?;
+			let Some(str) = ion::String::new(cx, text.as_str()) else {
+				return Err(ion::Error::new("Failed to allocate string", ion::ErrorKind::Normal));
```

Comment:
Using `.unwrap()` here is fine. If the engine runs out of memory, that's an uncatchable exception anyways.